keyboards/era/divine/info.json

```
@@ -0,0 +1,115 @@
+{
+    "manufacturer": "ERA",
+    "keyboard_name": "DIVINE 1.0.0",
+    "maintainer": "eerraa",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+    },
+    "matrix_pins": {
+        "cols": ["GP13", "GP12", "GP19", "GP20", "GP21", "GP22", "GP23", "GP24", "GP25", "GP26", "GP27", "GP28", "GP29", "GP18", "GP5", "GP6", "GP7"],
+        "rows": ["GP11", "GP10", "GP9", "GP8", "GP4"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x4552"
+    },
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "indicators": {
+        "caps_lock": "GP1",
+		"num_lock": "GP2",
+        "scroll_lock": "GP3"
+    },
+    "layouts": {
+        "LAYOUT_split_all": {
```

- remove comma in last `features` element
- change LAYOUT name to conform to QMK layout naming convention
```suggestion
        "nkro": true
    },
    "matrix_pins": {
        "cols": ["GP13", "GP12", "GP19", "GP20", "GP21", "GP22", "GP23", "GP24", "GP25", "GP26", "GP27", "GP28", "GP29", "GP18", "GP5", "GP6", "GP7"],
        "rows": ["GP11", "GP10", "GP9", "GP8", "GP4"]
    },
    "processor": "RP2040",
    "url": "",
    "usb": {
        "device_version": "1.0.0",
        "pid": "0x0002",
        "vid": "0x4552"
    },
    "bootmagic": {
        "matrix": [0, 1]
    },
    "indicators": {
        "caps_lock": "GP1",
		"num_lock": "GP2",
        "scroll_lock": "GP3"
    },
    "layouts": {
        "LAYOUT": {
```